cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/Conversion.java

```diff
@@ -278,6 +285,17 @@ private void assignNetworkProperties(Context context) {
         LOG.info("network forecastDistance : {}", context.network().getForecastDistance());
     }
 
+    private void addCgmesSvDescription(Network network) {
+        PropertyBags svDescription = cgmes.fullModel(CgmesSubset.STATE_VARIABLES.getProfile());
+        if (svDescription != null && !svDescription.isEmpty()) {
+            network.setProperty(CgmesNames.SCENARIO_TIME, svDescription.get(0).getId("scenarioTime"));
```

Comment:
if we add all these properties directly to the Network I think we should prefix all of them with "cgmes." or something like that, to avoid potential key collisions. 
All metadata related to CGMES should have that prefix.

Reply:
I think that it will not work to export an SV for the CGM model with the merging view. It will select the first property which key is "SCENARIO_TIME".

Reply:
I agree with Anne, I think this should be in an extension that will be built either during the merging (merge post-processor?) or by the user. I made the PR #1459 to create this extension.